AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java

```diff
@@ -254,6 +259,22 @@ private SaveNoteHandler saveNoteHandler() {
         return new SaveNoteHandler(this);
     }
 
+
+    @Override
+    public void onDeckSelected(@Nullable DeckSelectionDialog.SelectableDeck deck) {
+        if (deck != null) {
+            mCurrentDid = deck.getDeckId();
+            mNoteDeckSpinner.setSelection(mAllDeckIds.indexOf(deck.getDeckId()), false);
+        }
+    }
+
+    private void displayDeckOverrideDialog(Collection col) {
+        FunctionalInterfaces.Filter<Deck> nonDynamic = (d) -> !Decks.isDynamic(d);
+        List<DeckSelectionDialog.SelectableDeck> decks = DeckSelectionDialog.SelectableDeck.fromCollection(col, nonDynamic);
+        DeckSelectionDialog dialog = DeckSelectionDialog.newInstance("Deck Search", null, false, decks);
```

Comment:
This needs to be in a resource so our translators can translate it

```suggestion
        DeckSelectionDialog dialog = DeckSelectionDialog.newInstance("Deck Search", null, false, decks);
```